public/static/locales/en/launch.json

```diff
@@ -37,7 +37,7 @@
     "resourceRequirements": "Resource Requirements",
     "resourceRequirementsForStep": "Resource Requirements for Step {{step}}",
     "reviewAndLaunch": "Review and Launch",
-    "saveAsQuickLaunch": "Save as Quick Launch",
+    "saveAsSavedLaunch": "Save as Saved Launch",
```

Comment:
This one and the one above sound a little strange.  Not sure if `Create as Saved Launch` or just `Save Launch` might be better?

Reply:
I started to go with `Save`, but then with App Launch that looked a little confusing next to `Launch Analysis` on which action I should take next.
![image](https://user-images.githubusercontent.com/8909156/130159210-628f3714-f1c5-43aa-b778-9ea53902c716.png)

I'll go with `Create Saved Launch` for now. `Create` makes it sound like I'm generating something new where `Save` sounds like I'm saving my progress on this form while continuing to work on it.  But I'm still open to ideas on this one.